masonry_core/src/widgets/text_area.rs

```diff
@@ -279,6 +284,12 @@ impl<const EDITABLE: bool> TextArea<EDITABLE> {
         self
     }
 
+    /// Set the `InsertNewline` behavior
```

Comment:
Our style is to use full stops at the ends of doc comment sentences (usually; I see the one directly below this doesn't do so :sweat_smile: ). This applies to all the doc comments in this PR.